ci/build_wheel.sh

```diff
@@ -56,5 +56,9 @@ else
         -w "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}" \
         dist/*
 
-    RAPIDS_PY_WHEEL_NAME="${package_name}_${RAPIDS_PY_CUDA_SUFFIX}" rapids-upload-wheels-to-s3 python "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}"
+    if [[ ${package_name} == "libwholegraph" ]]; then
```

Comment:
cuGraph (and maybe other libraries) solve this differently, with a third argument `package_type`. I think that's a bit cleaner. See snippets linked below.

https://github.com/rapidsai/cugraph/blob/60a6990d9e5087e4579d5b462a6a48f40fa3d391/ci/build_wheel.sh#L8

https://github.com/rapidsai/cugraph/blob/60a6990d9e5087e4579d5b462a6a48f40fa3d391/ci/build_wheel_libcugraph.sh#L31

Reply:
thanks for the links, updated